table/builder.go

```diff
@@ -424,3 +437,69 @@ func (b *Builder) compressData(data []byte) ([]byte, error) {
 	}
 	return nil, errors.New("Unsupported compression type")
 }
+
+// type use 2 bits, now we have two unused type.
+const (
+	typeUint    = 0
```

Comment:
Can we remove the type?
For each field, the type is known.

Reply:
We have to support `metaDecoder.skip`